### PR TITLE
fixed this again

### PR DESCRIPTION
**Pull request by brettgaylor**
Doing this over again

### CODE REVIEW

File: templates/basic/saved-data.json [hunk 1]
```diff
@@ -61,17 +36,17 @@
                     "left": 26.96428571428571,
                     "top": 37.77777777777778,
                     "width": 50,
-                    "zindex": 999,
+                    "zindex": 1000,
                     "alignment": "center"
                 },
-                "track": "Layer",
+                "track": "0",
```
Comment (dseif) [inline templates/basic/saved-data.json:42]: Should this be `Layer 0` like it is in the first trackEvent?

File: templates/basic/saved-data.json [hunk 2]
```diff
@@ -89,55 +64,91 @@
...
                     "width": 50,
-                    "zindex": 999
+                    "zindex": 1000,
+                    "shadow": "",
+                    "shadowColor": "#444444",
+                    "background": "",
+                    "backgroundColor": "#888888"
                 },
-                "track": "Layer",
+                "track": "0",
```
Comment (dseif) [inline templates/basic/saved-data.json:73]: Same here

File: templates/basic/saved-data.json [hunk 3]
```diff
@@ -89,55 +64,91 @@
...
-                    "top": 38.41269841269842,
+                    "left": 29.82142857142857,
+                    "top": 33.33333333333333,
                     "width": 50,
-                    "zindex": 999
+                    "zindex": 998
                 },
-                "track": "Layer",
-                "name": "TrackEvent4"
+                "track": "2",
```
Comment (dseif) [inline templates/basic/saved-data.json]: If this doesn't matter just ignore it

File: templates/basic/saved-data.json [hunk 4]
```diff
@@ -89,55 +64,91 @@
...
+                "id": "TrackEvent3",
                 "type": "text",
                 "popcornOptions": {
-                    "start": 10.048350423988875,
-                    "end": 14.96402240301433,
+                    "start": 10.814615384615385,
+                    "end": 11,
                     "target": "video-container",
-                    "text": "Delete these events by pressing the delete key!\n",
+                    "text": "Delete these events after you&amp;#39;ve imported\nyour media!",
```
Comment (dseif) [inline templates/basic/saved-data.json]: replace `&amp;#39;` with an apostrophe

File: templates/basic/saved-data.json (hunk 3 above)
Comment (brettgaylor) [inline templates/basic/saved-data.json]: I dunno - these are just copied from the Remix URL...

**Pull request closed without merging**